spec/features/user_following_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+feature 'User following' do
+  scenario 'user follows and unfollows someone' do
+    alice = Fabricate(:user)
+    category = Fabricate(:category)
+    video = Fabricate(:video, category: category)
+    Fabricate(:review, user: alice, video: video)
+
+    sign_in
+    click_on_video_on_home_page(video)
```

Comment:
You could refactor the rest of this to get it to the same abstraction level as `sign_in` and `unfollow`.